rapid7_insightvm/komand_rapid7_insightvm/util/resource_requests.py

```diff
@@ -0,0 +1,203 @@
+from .shared_resources import *
+import json
+import requests
+import urllib3
+from komand.exceptions import PluginException
+from typing import NamedTuple, Collection
+
+
+class RequestResult(NamedTuple):
+    """
+    Class for data returned in paged InsightVM API requests
+    This is the Python 3.6 style for using named tuples with typing
+    """
+    page_num: int
+    total_pages: int
+    resources: dict
+
+
+class TestResult(NamedTuple):
+    """
+    Class for data returned in test method
+    This is the Python 3.6 style for using named tuples with typing
+    """
+    status: int
+    message: str
+
+
+class ResourceRequests(object):
+    """
+    Class for helper methods for making requests against the APIv3. A new instance should
+    be instantiated within the action/trigger being developed. New methods should be
+    created as instance methods to allow reference of the logger and session passed to
+    the __init__ function during instantiation.
+    """
+
+    # Static headers for all requests
+    _HEADERS = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+    }
+    _ENSURECONNECTIVITY = 'Ensure proper network connectivity between the InsightConnect orchestrator and the IVM console'
```

Comment:
```suggestion
    _ENSURECONNECTIVITY = 'Ensure proper network connectivity between the InsightConnect orchestrator and the InsightVM console'
```

Do we need ending punctuation here?